src/codegeneration/ModuleTransformer.js

```diff
@@ -87,7 +87,7 @@ export class ModuleTransformer extends TempVarTransformer {
   }
 
   getTempVarNameForModuleSpecifier(moduleSpecifier) {
-    let normalizedName = System.normalize(moduleSpecifier.token.processedValue, this.moduleName);
+    let normalizedName = $traceurRuntime.normalizeModuleName(moduleSpecifier.token.processedValue, this.moduleName);
```

Comment:
Can we import this from somewhere? I'm fine if it just a file forwarding `$traceurRuntime.normalizeModuleName` for now.

We should have a goal of not using `$traceurRuntime` in the compiler.